packages/@azure/servicebus/data-plane/test/sessionsTests.spec.ts

```diff
@@ -19,6 +19,7 @@ import {
 import {
   testMessagesWithSessions,
   testMessagesWithDifferentSessionIds,
+  testMessagesToSamePartitionsWithDifferentSessions,
```

Comment:
Will this work?

We are sending messages to 2 different sessions. Will running `peek(2)` give 2 as a result?
What if sessions are internally implemented in such a way that 2 messages with different sessionids but same partition key  dont land in the same partition?

Reply:
Yes, Sessionid takes precedence over partition key. If session ID is mentioned then they don't look at partition key and use session id to route the message. So If we use session, we don't need to pass partition key.